includes/reports/overview/services/class-sensei-reports-overview-service-courses.php

```diff
@@ -25,13 +25,12 @@ public function __construct() {
 	/**
 	 * Get total average progress value for courses.
 	 *
+	 * @param array $courses_ids Courses ids.
+	 * @return float total average progress value for all the courses.
 	 * @since  4.5.0
 	 * @access public
-	 *
-	 * @return float total average progress value for all the courses.
 	 */
-	public function get_total_average_progress(): float {
-		$courses_ids = $this->get_all_courses_ids();
+	public function get_total_average_progress( array $courses_ids ): float {
```

Comment:
I think we should keep the naming consistent and pluralize only the last noun `$courses_ids` -> `$course_ids`. Same reasoning as in https://github.com/Automattic/sensei/pull/5091#discussion_r869161846.

Reply:
https://github.com/Automattic/sensei/pull/5091/commits/7a39446ccdc33bfd2055e71f3ccc4d77bed6e064